mkdocs.yml

```diff
@@ -221,7 +223,7 @@ nav:
 markdown_extensions:
     - toc:
         permalink: 
-        toc_depth: 3
+        toc_depth: 4
```

Comment:
This setting impacts all the doc site, I'm not sure it's good to change it. We should adapt the new content to fit in 3 levels.